updating/understanding-openshift-upgrade-duration.adoc

```diff
@@ -0,0 +1,91 @@
+:_content-type: ASSEMBLY
+[id="understanding-openshift-upgrade-duration"]
+= Understanding openshift upgrade duration
+include::_attributes/common-attributes.adoc[]
+:context: understanding-openshift-upgrade-duration
+
+toc::[]
+
+{product-title} 4 upgrade duration varies based on the deployment topology. This section describes the approximate time duration of each component that follows through during the cluster update. It also explains the factors that affect the upgrade duration and the applicable {product-title} parameters.
+
+== Prerequisites
+* You are familiar with xref:../architecture/architecture.adoc#architecture[OpenShift Container Platform architecture] and xref:../updating/understanding-openshift-updates.adoc#understanding-openshift-updates[OpenShift Container Platform updates].
+
+== Cluster upgrade duration
+
+Historical upgrade duration of similar clusters provides the best estimate for the future upgrades. However, if the historical data is not available, you can refer the below table to estimate the time duration of cluster update.
+
+The following table summarizes the approximate time duration of cluster update components:
+
+[options="header"]
+|===
+|Component Name |Time Duration (in minutes)
+
+|Cluster Version Operator (CVO)
+|45-120
+
+|Machine Config Operator (MCO)
+|5 per node
+|===
+
+=== Cluster Version Operator target upgrade payload deployment
+The Cluster Version Operator (CVO) unpacks the target upgrade release image and applies to the cluster. This process might take somewhere between 45 to 120 minutes.
+
+=== Machine Config Operator
+The Machine Config Operator (MCO) applies a new machine configuration to each control plane and worker node. During this process, the MCO performs the following tasks on each node of the cluster:
+
+* Node draining
+* OS update staging
+* Node rebooting
+
+The control plane node upgrade duration is more predictable and oftentimes shorter than worker nodes since it does not include large workloads. You can upgrade the worker nodes in parallel by setting the `maxUnavailable` field to greater than 1 in the machine configuration pool. The MCO cordons the number of nodes specified in `maxUnavailable` and marks them unavailable for upgrade.
+
+[IMPORTANT]
+====
+With {product-title} 4 upgrades, an operating system lifecycle management and maintenance is made as a part of the cluster upgrade.  This only applies to {op-system-first} nodes.
+====
+
+The time to complete this process depends on many factors including the node and infrastructure configuration. This process might take 5 minutes to complete per node.
```

Comment:
"5 minutes" to "5 minutes or more" or something?  5 minutes is the fastest we can roll a node.  But the time it takes to roll a node is unbounded, because maybe it's running a PDB-guarded workload that refuses to drain.

Reply:
"might take more than 5 minutes" opens the door to >5, but does not close the door on <5.  We want to express >=5m, so folks understand why we use 5m when calculating the minimum update duration.

Reply:
@wking I agree with you. I have made the following changes:

> This process might take 5 or more minutes to complete per node.